.travis.yml

```diff
@@ -23,23 +23,22 @@ before_install:
   - if [ "$TRAVIS_BRANCH" = "master" -a "$TRAVIS_PULL_REQUEST" = "false" -a "$DEPLOY" = "true" ]; then bash scripts/decrypt-keys.sh; fi
 
 install:
-  - rvm use 2.2.5 --install --fuzzy
+  - rvm use 2.4.3 --install --fuzzy
   - gem update --system
   - gem install sass
-  - gem install jekyll -v 3.2.1
+  - gem install jekyll -v 3.6.2
 
 cache:
   directories:
-  - $HOME/.sbt/0.13/dependency
   - $HOME/.sbt/boot/scala*
   - $HOME/.sbt/launchers
   - $HOME/.ivy2/cache
   - $HOME/.nvm
 
 script:
-  - ./sbt ++$TRAVIS_SCALA_VERSION clean coreJS/test lawsJS/test
-  - ./sbt ++$TRAVIS_SCALA_VERSION clean coverage coreJVM/test lawsJVM/test coverageReport
-  - ./sbt ++$TRAVIS_SCALA_VERSION docs/makeMicrosite
+  - ./sbt "++${TRAVIS_SCALA_VERSION}!" clean coreJS/test lawsJS/test
+  - ./sbt "++${TRAVIS_SCALA_VERSION}!" clean coverage coreJVM/test lawsJVM/test coverageReport mimaReportBinaryIssues
```

Comment:
@johnynek this is why the build is failing now I think! #213 removed it due to the cats upgrade, but we're going to have to leave it out in any PRs that include that, until there's a new release to compare against

Reply:
Yep. I was the one who said we needed to remove but then manually added it back like a fool.